python/cudf/cudf/window/rolling.py

```diff
@@ -110,6 +110,24 @@ def __init__(self, obj, window, min_periods=None, center=False):
         self.center = center
         self._normalize()
 
+    def __getattr__(self, key):
+        if key == "obj":
+            raise AttributeError()
+        return self.obj[key].rolling(
+            window=self.window,
+            min_periods=self.min_periods,
+            center=self.center,
+        )
+
+    def __getitem__(self, arg):
+        if isinstance(arg, tuple):
+            arg = list(arg)
```

Comment:
Does this need to handle accessing an index / MultiIndex?

Also, I could imagine a user feeding something returned from `.column` which will be a Pandas object or a numpy array here and we should handle it. Maybe it's time to create a utility for handling arbitrary host list-likes?

Reply:
Connected offline, apparently the access pattern would be `rolling([[(0, 1)]])` for accessing a MultiIndex so this is as expected.